### ISSUE TITLE
No way to search and replace

### ISSUE THREAD

**Issue by httpsterio** [Zettlr/Zettlr#19]
Currently I can't find the functionality to search and replace with Zettlr. Maybe it's possible somehow through Regexp but alas, I have no grasp of those :D 

Maybe CMD/CTRL+F to search normally (visualize how many results somewhere).

CMD+Shift+F or CMD+Shift+H (as in VSCode) to open Search and Replace.

Is this hard to implement? If you point me where the search functionality is defined, I might try to do a pull request. 

Also, are you in need of localization help?

**Comment by nathanlesage** [comment]
Hey @httpsterio,

I've moved the search/replace functionality from 0.16 -> 0.17 to a dedicated popup in the toolbar (the magnification glass). The shortcut Cmd/Ctrl+F should still work, though. I've combined both search and replace in this popup, although my own implementation is missing some functionality, which will come in the course of the next versions (such as counting matches and visualising their position on the scrollbar).

(But thanks for directing my attention to the search popup, I've just uncovered a bug!)

So, can you confirm that it works or is there another bug that simply doesn't occur on macOS?

Further, concerning the localisation: If you want to translate, I'd be more than happy to count on your  support!

Cheers

**Comment by httpsterio** [comment]
Yeah apparently in 0.17 search and replace is working as expected, thanks for that!

One thing that I noticed is that 0.17 doesn't hilight all found whereas 0.16 did. If possible, I think it should be an option. Also, after searching and replacing, if I want to undo it, each undo will only undo one change, but if I searched and replaced several items, it'll need several undos to undo all of those.

Is this working as expected?

**Comment by nathanlesage** [comment]
The highlighting of all search results will be imlemented either in 0.17.1 or 0.17.2, I've got this on my radar already!

As for the history events: If you repeatedly replace single findings, yes, these should generate distinct history events. As for the "replace all" option, it should only generate one distinct history event. Currently, it also generates a lot of different events, which I will change in one of the next versions (after, of course, the highlighing has been implemented)!
